jarbas/core/migrations/0024_add_available_in_latest_dataset_field_to_reimbursement.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
@cacarrara this is how django stores database migrations. This `migrations` directory is always present whenever a new django app is created.